### PR TITLE
Reimplementing the exisiting Automation cmdlets and adding new cmdlets

### PR DESCRIPTION
**Pull request by elvg**
We are updating our existing cmdlets because of our new Automation MAML  (not released yet) and adding new cmdlets for other Automation Resources likes certificates, variables, modules, etc.

Note that our PR hasn't been merged yet, that's why we haven't changed the MAML package version in our project. You can see the PR for azure-sdk-for-net from here: https://github.com/Azure/azure-sdk-for-net/pull/948

Please Do Not merge until we change the MAML package version.

Thanks.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@elvg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (elvg). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@elvg Please fix test failures

**Comment by markcowl** [comment]
@elvg please fix build issue - looks like you need to have the correct version of automation cmdlets?

**Comment by elvg** [comment]
@markcowl since SDK hasn't merged yet, we don't have our automation SDK released with that version. We are still waiting it to be merged in azure-sdk-for-net.

**Comment by markcowl** [comment]
@elvg, the azure-sdk-for-net PR is merged

File: src/Common/Commands.ScenarioTest/Resources/Automation/AutomationTests.ps1 [hunk 1]
```diff
@@ -17,7 +17,8 @@
 Checks whether the first string contains the second one
 #>
 
-$accountName='AutomationAccount'
+$accountName='safeer'
```
Comment (ogail) [inline src/Common/Commands.ScenarioTest/Resources/Automation/AutomationTests.ps1]: nit: it's little bit awkward to have account name with a real person. Maybe `automatedAccount` is better?`

Comment (elvg) [inline src/Common/Commands.ScenarioTest/Resources/Automation/AutomationTests.ps1]: thanks for catching this but we would like to change it in next release.

**Comment by elvg** [comment]
Thanks!

**Pull request merged**